python/fastexcel/__init__.py

```diff
@@ -61,22 +61,39 @@ def sheet_names(self) -> list[str]:
         """The list of sheet names"""
         return self._reader.sheet_names
 
-    def load_sheet_by_name(self, name: str) -> ExcelSheet:
+    def load_sheet_by_name(
+        self,
+        name: str,
+        header_row: None | int = 0,
```

Comment:
```suggestion
        self,
        name: str,
        /,
        header_row: None | int = 0,
```

Making the other arguments named-only

Reply:
Uh crap, I meant `*` of course! I wanted named only because I'm afraid the number of parameters will keep growing